static_content/css/main.css

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Unnecessary file for the PR (which also contains a sass error by the looks of it)